README.md

```diff
@@ -35,7 +35,8 @@ You will need an authorization token to create a channel on Kolibri Studio. In o
 
 ### Step 2: Initializing the Channel ###
 
-To run the Ricecooker, you must include a `construct_channel` method in your sushi chef file that returns a ChannelNode object. This function will be responsible for building the structure of your channel.
+To run the Ricecooker, you must include `create_channel` and `construct_channel` methods in your sushi chef file.
```

Comment:
It could. But the idea of create_channel is just to create only the channel node (root node) as fast as possible, this is used to configure the sushi bar client as early as possible in upload_channel (this helps, among other things, to start reporting remote logs very early in the workflow). We discussed this with Jamie during last week meeting. construct_channel is the same as before, it can be a heavy weight call.